README.md

```diff
@@ -26,6 +26,8 @@ Many different LED-strips can be used with this program. The only requirement is
 
 I used the [IKEA Dioder](https://www.ikea.com/us/en/catalog/categories/series/25230/) LED-strip which I connected to my Raspberry Pi 3 model B+ by following [this](https://dordnung.de/raspberrypi-ledstrip/) guide.
 
+To use a WS281X led strip (Neopixels) you need to set the `is_active` value in `config.ini` to `True`, the `led_count` value to the number of leds in your strip and the `led_pin` to the GPIO pin you connected the data input of your led strip to. The other values under `[WS281X]` are optional and set as default. You can refer to [this](https://tutorials-raspberrypi.com/connect-control-raspberry-pi-ws2812-rgb-led-strips/) guide for additional help but don't run the setup commands there as the required library is now installed by using `pip` rather than the manual way in the guide.
```

Comment:
All but the guide (and some text around it) should be in the "Setup config file" section instead.